content_script.js

```diff
@@ -176,6 +166,31 @@ class HTMLSerializer {
       return fullyQualifiedName + '.' + index; 
     }
   }
+
+  /**
+   * Calculate the correct quotes that should be used given how many parent
+   * windows a given window has.
+   *
+   * @param {number} depth The number of parent windows.
```

Comment:
Instead of "number of parent windows", maybe "nesting depth of this window in the frame tree"